content/sips/sip-120.md

```diff
@@ -1,7 +1,7 @@
 ---
 sip: 120
 title: TWAP Exchange Function
-status: Approved
+status: Feasibility
```

Comment:
The is actually ready to present to the council. Can we slot it in for next week?